packages/runtime/src/useCases/consumption/attributes/GetRepositoryAttributes.ts

```diff
@@ -26,15 +38,16 @@ export class GetRepositoryAttributesUseCase extends UseCase<GetRepositoryAttribu
     }
 
     protected async executeInternal(request: GetRepositoryAttributesRequest): Promise<Result<GetRepositoryAttributesResponse>> {
-        const query: any = {
-            shareInfo: { $exists: false }
-        };
+        const query: GetAttributesRequestQuery = request.query ?? {};
+        const flattenedQuery = flattenObject(query);
+        const dbQuery = GetAttributesUseCase.queryTranslator.parse(flattenedQuery);
```

Comment:
OH, I just stumbled upon this, please copy the query translator. We shouldn't use query translators from other UseCases. Is this the case for others??

Reply:
Yea, it is used in 4 use cases

![image](https://github.com/nmshd/runtime/assets/9070136/3d7c5927-b929-4ebb-8e9a-fac0e1124ca7)

Reply:
I have something in the back of my head about that we shouldn't use this `flattenedQuery` method, that it's more a relict of past and that we're supposed to use query objects instead of the flat query strings. @jkoenig134 could you comment on that? This may just be an issue of copy-pasting stale code.

Reply:
I think it's old, but still safe to use. I'd say we can use it like this for now.